src/patterns/scheduled-ecs-task.test.ts

```diff
@@ -17,13 +17,14 @@ const makeVpc = (stack: GuStack) =>
 describe("The GuScheduledEcsTask pattern", () => {
   it("should use the specified schedule", () => {
     const stack = simpleGuStackForTesting();
-
+    const vpc = makeVpc(stack);
     new GuScheduledEcsTask(stack, "test", {
       schedule: Schedule.rate(Duration.minutes(1)),
       containerConfiguration: { id: "node:10", type: "registry" },
       monitoringConfiguration: { noMonitoring: true },
-      vpc: makeVpc(stack),
+      vpc,
       app: "ecs-test",
+      subnets: vpc.privateSubnets,
```

Comment:
IIUC the snapshot tests haven't changed, and are still passing. I can't quite work out why 🤔. Does it indicate that we don't have enough test coverage? Or that we don't need this change after all?

Reply:
Good point. I've added a test here which I think might help a bit https://github.com/guardian/cdk/pull/2477/commits/116280e45f94163015ab34f1f8c96ef12632d838

The ecs task itself hasn't changed much - it's just that `subnets` is now a required prop - whereas previously it would be inferred from context. I think in the tests the 'inferring from context' maybe just worked and picked the private subnets, which we're now explicitly setting as the subnets